spec/outputs/datadog_metrics_spec.rb

```diff
@@ -1 +1,86 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/datadog_metrics"
+
+describe LogStash::Outputs::DatadogMetrics do
+  let(:http_instance) { double('net_http_instance') }
+  let(:request) { double('net_http_request') }
+  let(:response) { double('net_http_response') }
+
+  let(:json_obj) { double('json_obj') }
+  let(:logger) { double('logger') }
+  let(:ddtags) { ['foo', 'bar'] }
+  let(:event_hash) do
```

Comment:
why not just calling this event? is there any other representation of the event so we need to include the word hash?